plugin.video.fsgo/addon.xml

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.fsgo"
+       version="1.1.5"
+       name="FOX Sports GO"
+       provider-name="emilsvennesson">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="script.module.requests" version="2.9.1"/>
+    <import addon="script.module.m3u8" version="0.2.10"/>
+    <import addon="script.module.iso8601" version="0.1.11"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="default.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <description lang="en">Watch live sports and shows from FOX Sports GO.[CR][CR]Please note that this add-on requires you to authenticate to one of FOX Sports GO's participating TV providers.</description>
```

Comment:
It would be cool if you could chance en to en_GB as it is the new default. This doesn't apply to the language tag.